includes/Admin/Helpers/FormFieldsHelper.php

```diff
@@ -781,7 +781,7 @@ public function init_share_of_checkout_field( $default_settings ) {
 				'title'       => '<hr>' . __( '→ Increase your performance & get insights !', 'alma-gateway-for-woocommerce' ),
 				'type'        => 'title',
 				'description' => wp_kses_post(
-					__( 'By accepting this option, enable Alma to analyse the usage of your payment methods, get more informations to perform and share this data with you.', 'alma-gateway-for-woocommerce' ) .
+					FormFieldsHelper . php__( 'By accepting this option, enable Alma to analyse the usage of your payment methods, get more informations to perform and share this data with you.', 'alma-gateway-for-woocommerce' ) .
```

Comment:
What is `php` before the function `__()` ?

Reply:
done